Library/Formula/wine.rb

```diff
@@ -67,10 +66,11 @@ def require_universal_deps?
   depends_on 'libtiff'
   depends_on 'sane-backends'
   depends_on 'libgsm' => :optional
+  option "enable-win64",
+         "Build with win64 emulator (won't run 32-bit binaries.)" if build.head?
```

Comment:
Then that's a separate bug. It still needs to go in the block. Using `build.head?` in the DSL is disallowed going forward.